Makefile

```diff
@@ -319,7 +319,7 @@ $(foreach prog,$(PROGRAMS),$(eval $(call BUILD_ASSEMBLY,$(prog))))
 #
 default: core
 
-core: game platforms mods utility server
+core: dependencies game platforms mods utility server
```

Comment:
I don't think that's necessary. As you said this is just a task runner and the latter targets depend on the dependencies target.